pymango/client.py,sha256=sNi1YZ2VyAc6F1BfZ0C0DD1XR5SCQXafNsYigH6vTm4,2621
pymango/error.py,sha256=GF0Ttji0_bp16YT7QCCVR7f3RCb7WU4L659LhXaAp6s,1025
pymango/resources.py,sha256=-MzLCUCKvPuiCRfk7gMddRYYOldrZyG-yZmKM-N1xp0,4129
pymango/__init__.py,sha256=5cAWAkWar0GqFnVaKyngUDq5BgzwSEJy6-l-o6U_34o,379
pymango-0.1.1.dist-info/DESCRIPTION.rst,sha256=hW6iDVIMEDli7-k5VbbjWMXiThiV0SeZhwbFyOJ1D_k,58
pymango-0.1.1.dist-info/METADATA,sha256=60jBZRqfd_bUzIZZFhFHWwuLS2z2FNnHkW3GDSnkwWQ,372
pymango-0.1.1.dist-info/metadata.json,sha256=BMYKarMrtlNXBxzp9mYiSlCexMedFyPoaQGtMwAWz-A,541
pymango-0.1.1.dist-info/RECORD,,
pymango-0.1.1.dist-info/top_level.txt,sha256=Nj5RXpkk-B664AFnsznYwXUPrMPk3bPP3SJ4JKkwmOQ,8
pymango-0.1.1.dist-info/WHEEL,sha256=R7GZBaynU7P7eIwbsGv7F-X8eM9JWO6nKzGDfrA5_lo,97
pymango/__pycache__/__init__.cpython-34.pyc,,
pymango/__pycache__/error.cpython-34.pyc,,
pymango/__pycache__/resources.cpython-34.pyc,,
pymango/__pycache__/client.cpython-34.pyc,,
